app/assets/javascripts/services_renderers_enablers.js.es6

```diff
@@ -34,6 +34,16 @@ function voteIdeaQuality(inputName){
   });
 }
 
+function updateIdeaContent(element, $object, updateContent, ideaId, errorMessage) {
```

Comment:
This feels like a lot of params to try to keep in order when calling the function in the future. Maybe stuff into a params hash so that order doesn't matter?